t/integration/test_canvas.py

```diff
@@ -952,6 +953,26 @@ def test_chord_in_chain_with_args(self, manager):
         res1 = c1.apply(args=(1,))
         assert res1.get(timeout=TIMEOUT) == [1, 1]
 
+
+    @pytest.mark.xfail(reason="Issue #6200")
+    def test_chain_in_chain_with_args(self):
+        try:
+            manager.app.backend.ensure_chords_allowed()
+        except NotImplementedError as e:
+            raise pytest.skip(e.args[0])
+
+        c1 = chain(  # NOTE: This chain should have only 1 chain inside it
+            chain(
```

Comment:
So this doesn't fold into one chain anymore?

Reply:
Not too sure - but considering it works for redis - I guess it is fine. It just seems to be an issue with the `.apply()`